### DISCUSSION TITLE
Stable support for dart records

### DISCUSSION THREAD

**Discussion by listepo** [simolus3/drift#2762] [Q&A]
Dart records now are stable, when we see that in drift?

**Comment by simolus3** [comment]
You can use records as row classes for drift by using `@UseRowClass` annotation - `@UseRowClass(Record)` makes drift generate a suitable record, but you can also specify an existing shape like

```dart
typedef User = ({String name, int id});

@UseRowClass(User)
class Users extends Table {
  // ...
}
```

What kind of additional record support were you looking for?
